### DISCUSSION TITLE
Resign from CHANGELOG.md to specify such info in releases

### DISCUSSION THREAD

**Discussion by DevHusariaSolutions** [solidjs/solid#1070] [Ideas]
This file is outdated and updateing that forces team to do some extra steps.
It's not consistent, because forgetting about extending that file requires change in newer version of code, which should extend such description by "added missing changelog info".

Such description can be added in Realeses section, like in NextJS repo:
https://github.com/vercel/next.js/releases

**Comment by ryansolid** [comment]
We do add them to the releases as well. I usually use the changelog while the release is in progress. And then copy them over for the final release notes. I still do a fairly manual process so it works for me. But were that to change I can see it no longer having use.
